pipelines/rh-advisories/rh-advisories.yaml

```diff
@@ -407,6 +407,8 @@ spec:
       workspaces:
         - name: data
           workspace: release-workspace
+      runAfter:
```

Comment:
This is redundant, we already rely on create-pyxis-image.results in the parameters